src/js/yeldaWebchatInjector.js

```diff
@@ -775,40 +775,109 @@ class YeldaChat {
     return isFound
   }
 
+  /**
+   * Get the app env from the window url
+   * @param {String} href 
+   * @returns {String} env
+   */
+  getAppEnv(href) {
+    if (config.REGEX_HOST.STAGING.test(href)) {
+      return 'staging'
+    }
+    if (config.REGEX_HOST.LOCALHOST.test(href)) {
+      return 'local'
+    }
+    return 'prod'
+  }
```

Comment:
:gear: let's not guess the env from the host (for example, for prod debug i quite often connect my local chat plugin to staging or prod), but from 
1. assistantUrl (see `setAssistantUrl`)
- if the endpoint is using staging, the env should be staging
- if the endpoint is production, the env should be production
- if the endpoint is local, then the env should set by the developer with a parameter
```js
yeldaChat.init({settingId: xxxx, env:'xx'}),
```
with default (if no env provided) to production, so no compatibility issue

:writing_hand: this 'prod' value should come from config

:warning: are you sure about 'prod' ? 
The redis prefix is `prefix: process.env.NODE_ENV`, and in prod, this value is 'production'
![image](https://user-images.githubusercontent.com/5340081/150316488-253bbb86-6ffe-4469-8c74-4ac1912f2167.png)